src/model/rules.js

```diff
@@ -65,6 +65,15 @@ const createEngine = () => {
   engine.addOperator('deepEqual', (factValue, ruleValue) => {
     return deepEqual(factValue, ruleValue)
   })
+  engine.addOperator('isString', (factValue, ruleValue) => {
```

Comment:
Is it possible to add documentation on how to use the rules for the quoting-service?